doc/usecases/plugins/spec_globbing.md

```diff
@@ -0,0 +1,51 @@
+# Use Case: Specification for application communicating with multiple services
+
+## Summary
+
+- **Scope:** Configuration
+- **Level:** Developer Goal
+- **Actors:** Application developer
+- **Brief:** The user wants to specify the port and url for an unknown number of servers, which the application shall use.
+
+## Scenarios
+
+- **Precondition:** kdb is installed.
+- **Main success scenario:**
+  - Specification is written and stored system-wide or user specific.
+  - All services which are added in the configuration need to adhere to this configuration specification.
+- **Alternative scenario:** None.
+- **Error scenario:**
+  - In case services have overlapping configuration an error is yielded.
+  - A sample for such overlapping would be `service/#/port` and `service/_/port`.
```

Comment:
This comes from nowhere. First the main success scenario should introduce one way, then alternative/error scenarios can show another way.